cms/templates/widgets/header.html

```diff
@@ -188,14 +187,39 @@ <h2 class="info-course">
     </div>
 
     <div class="wrapper wrapper-r">
-      % if user.is_authenticated():
+      % if static.show_language_selector():
+        <% languages = static.get_released_languages() %>
+        % if len(languages) > 1:
+        <nav class="user-language-selector">
+          <form action="/i18n/setlang/" method="post" class="settings-language-form" id="language-settings-form">
+              <input type="hidden" id="csrf_token" name="csrfmiddlewaretoken" value="${csrf_token}">
+              % if static.is_user_authenticated():
+              <input title="preference api" type="hidden" id="preference-api-url" class="url-endpoint" value="${reverse('preferences_api', kwargs={'username': user.username})}" data-user-is-authenticated="true">
```

Comment:
Let's remove the `id` attributes of these hidden inputs if they are no longer used.